### PR TITLE
Use the node versions as specified (Linux)

### PR DESCRIPTION
**Pull request by kneth**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -110,9 +110,9 @@ stage('test') {
   for (def nodeVersion in nodeVersions) {
     parallelExecutors["macOS node ${nodeVersion} Debug"]   = testMacOS("node Debug ${nodeVersion}")
```
Comment (kraenhansen) [inline Jenkinsfile]: Could / should the `testMacOS` have the same arguments as `testLinux`?

File: scripts/test.sh [hunk 2]
```diff
@@ -353,9 +353,20 @@ case "$TARGET" in
 
   check_test_results ReactTests
   ;;
+
+"node-test-runners")
```
Comment (kraenhansen) [inline scripts/test.sh]: Should this be "test-runners"?
Is this different from https://github.com/realm/realm-js/blob/master/scripts/test.sh#L397-L401?

Comment (kneth) [reply scripts/test.sh]: I'll remove it and adjust `Jenkinsfile`.

**Comment by kneth** [comment]
I am merging this after manually inspecting that the specified node version is actually used on Linux.

**Pull request merged**